src/pages/UniversalSwap/Swap/hooks/useSimulate.ts

```diff
@@ -22,20 +22,21 @@ export const useSimulate = (
   routerClient: OraiswapRouterReadOnlyInterface,
   initAmount?: number
 ) => {
-  const [[fromAmountToken, toAmountToken], setSwapAmount] = useState([initAmount || 0, 0]);
+  const [[fromAmountToken, toAmountToken], setSwapAmount] = useState([initAmount || null, 0]);
 
   const { data: simulateData } = useQuery(
     [queryKey, fromTokenInfoData, toTokenInfoData, fromAmountToken],
     () => {
-      console.log('original from info chain id: ', originalFromTokenInfo.chainId);
       return handleSimulateSwap({
         originalFromInfo: originalFromTokenInfo,
         originalToInfo: originalToTokenInfo,
         originalAmount: fromAmountToken,
         routerClient
       });
     },
-    { enabled: !!fromTokenInfoData && !!toTokenInfoData && fromAmountToken > 0 }
+    {
+      enabled: !!fromTokenInfoData && !!toTokenInfoData && fromAmountToken > 0
```

Comment:
[Low priority] I'd love the whole comparison wrapped in a variable in context. may be `hasFromTokenData`